src/quic/node_quic_socket.cc

```diff
@@ -734,68 +719,47 @@ BaseObjectPtr<QuicSession> QuicSocket::AcceptInitialPacket(
     ssize_t nread,
     const uint8_t* data,
     const SocketAddress& local_addr,
-    const sockaddr* remote_addr,
+    const SocketAddress& remote_addr,
     unsigned int flags) {
   HandleScope handle_scope(env()->isolate());
   Context::Scope context_scope(env()->context());
   ngtcp2_pkt_hd hd;
   QuicCID ocid;
-  uint64_t initial_connection_close = NGTCP2_NO_ERROR;
 
   // If the QuicSocket is not listening, the paket will be ignored.
   if (!is_flag_set(QUICSOCKET_FLAGS_SERVER_LISTENING)) {
     Debug(this, "QuicSocket is not listening");
     return {};
   }
 
-  // Perform some initial checks on the packet to see if it is an
-  // acceptable initial packet with the right QUIC version.
-  switch (QuicSession::Accept(&hd, version, data, nread)) {
-    case QuicSession::InitialPacketResult::PACKET_VERSION:
-      SendVersionNegotiation(
-          version,
-          dcid,
-          scid,
-          local_addr,
-          remote_addr);
-      return {};
-    case QuicSession::InitialPacketResult::PACKET_RETRY:
-      Debug(this, "0RTT Packet. Sending retry.");
-      SendRetry(version, dcid, scid, local_addr, remote_addr);
+  switch (ngtcp2_accept(&hd, data, static_cast<size_t>(nread))) {
+    case 1:
+      // Send Version Negotiation
+      SendVersionNegotiation(version, dcid, scid, local_addr, remote_addr);
+      // Fall through
+    case -1:
+      // Either a version negotiation packet was sent or the packet is
+      // an invalid initial packet. Either way, there's nothing more we
+      // can do here.
       return {};
-    case QuicSession::InitialPacketResult::PACKET_IGNORE:
-      return {};
-    case QuicSession::InitialPacketResult::PACKET_OK:
-      break;
-  }
+  };
```

Comment:
```suggestion
  }
```

(I assume this was a typo?)

Reply:
Yep :-)